dotcom-rendering/src/components/EuropeLandingModal.importable.tsx

```diff
@@ -0,0 +1,344 @@
+import { css } from '@emotion/react';
+import { getCookie, setCookie } from '@guardian/libs';
+import {
+	body,
+	from,
+	headline,
+	palette,
+	space,
+} from '@guardian/source-foundations';
+import {
+	Button,
+	Radio,
+	RadioGroup,
+	SvgCross,
+} from '@guardian/source-react-components';
+import { useState } from 'react';
+import type { EditionId } from '../lib/edition';
+import { getEditionFromId } from '../lib/edition';
+import { guard } from '../lib/guard';
+import { useOnce } from '../lib/useOnce';
+import { SvgFlagsInCircle } from './SvgFlagsInCircle';
+
+const modalShownKey = 'gu.euModalShown';
+
+// Have not used margin as it will change the backdrop
+const dialogStyles = css`
+	&::backdrop {
+		background: rgba(18, 18, 18, 0.7);
+		padding: 10px;
+	}
+	&[open] {
+		display: flex;
+	}
+	max-width: calc(100vw - 20px);
+	padding: 0;
+	width: 620px;
+	justify-content: space-between;
+	background: white;
+	flex-direction: row;
+	top: 0;
+	position: fixed;
+	border: none;
+	z-index: 1000;
```

Comment:
Might be worth adding a comment here
```suggestion
    /* We cant use getZIndex() here as this dialog has to overlap content on the page that does not follow our getZIndex standard (thrashers) */
	z-index: 1000;
```